pages/1.11/installing-upgrading/cloud/azure/index.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: layout.pug
+excerpt:
+title: Installing on Azure
+navigationTitle: Installing on Azure
+menuWeight: 5
+
+oss: true
+---
+
+This document explains how to install DC/OS 1.11 using the Azure Resource Manager templates.
+
+**Tip:** To get support on Azure Marketplace-related questions, join the Azure Marketplace [Slack community](http://join.marketplace.azure.com).
+
+**Important:** Upgrades are not supported with this installation method.
```

Comment:
This warning should be styled the same as the one for aws. (using message type = warning).